src/content/docs/en/guides/actions.mdx

```diff
@@ -581,3 +581,21 @@ export const server = {
 }
 ```
 :::
+
+## Call actions from Astro components and server endpoints
+
+You may want to call actions directly from Astro component scripts or server endpoints. This is common when reusing logic from your actions in other server code.
+
+Call actions using the `Astro.callAction()` wrapper (or `context.callAction()` when using a [server endpoint](http://localhost:4321/en/guides/endpoints/#server-endpoints-api-routes)), passing the action as the first argument and any input parameters as the second argument. This returns the same `data` and `error` objects you receive when calling actions on the client:
```

Comment:
It's quite optimistic to assume that every user always has a development server of the documentation up and running. ;)

```suggestion
Call actions using the `Astro.callAction()` wrapper (or `context.callAction()` when using a [server endpoint](/en/guides/endpoints/#server-endpoints-api-routes)), passing the action as the first argument and any input parameters as the second argument. This returns the same `data` and `error` objects you receive when calling actions on the client:
```